roles/pulp-webserver/tasks/main.yml

```diff
@@ -1,4 +1,20 @@
 ---
+- name: Find pulpcore directory
+  find:
+    paths: '{{ pulp_install_dir }}'
+    recurse: true
+    file_type: directory
+    patterns: pulpcore
+  register: result
+
+- name: Assert one directory has been found
+  assert:
+    that: '{{ result.matched }} == 1'
+
+- name: Set fact referencing Pulp's static files directory
+  set_fact:
+    pulp_webserver_static_dir: '{{ (result.files[0].path~"/../rest_framework") | realpath }}'
```

Comment:
pretty sure this won't actually work on system python :)

Reply:
shouldn't this be something like `/var/lib/pulp/static`?

Reply:
`/var/lib/pulp`, but this doesn't work because that folder is not readable by the `nginx` user.

Reply:
seems to be worked on in https://github.com/pulp/ansible-pulp/pull/178, so won't touch it for now myself